### PR TITLE
Add update view method and format project

### PR DESCRIPTION
**Pull request by s25g5d4**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (marcusschiesser) [inline LICENSE]: Why did you delete the license file?

File: src/glasnostic-api.ts [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+            view.name = name;
+        }
+        if (!isNil(source)) {
+            view.clients = source;
+        }
+        if (!isNil(destination)) {
+            view.services = destination;
+        }
+        if (!isNil(policies) && !isNil(view.policies)) {
+            view.policies = Policies.merge(view.policies, policies);
```
Comment (marcusschiesser) [inline src/glasnostic-api.ts]: Why do we need to merge?

File: LICENSE (hunk 1 above)
Comment (s25g5d4) [reply LICENSE]: so this is intended? I didn't know you want to have it MIT licensed

File: src/glasnostic-api.ts (hunk 2 above)
Comment (s25g5d4) [reply src/glasnostic-api.ts]: because view commit would override all value. we need to merge original view with new value to avoid making unwanted change.

File: package.json [hunk 3]
```diff
@@ -2,18 +2,26 @@
   "name": "glasnostic-api",
   "version": "1.0.0",
   "description": "",
-  "main": "glasnostic-api.js",
+  "files": ["dist/**/*"],
+  "main": "dist/glasnostic-api.js",
   "scripts": {
     "build": "tsc",
```
Comment (marcusschiesser) [inline package.json]: merge `build` and `build:example`

File: package.json [hunk 4]
```diff
@@ -2,18 +2,26 @@
...
+    "build:example": "tsc -p example/tsconfig.json",
+    "run-example:create-view": "node example/build/example/create-view.js",
```
Comment (marcusschiesser) [inline package.json]: move to `README`

File: README.md [hunk 5]
```diff
@@ -12,17 +12,15 @@ See `example.ts` for usage.
...
-```
-
-output will be in the `dist` folder.
+output will be in the `dist` folder. Examples will be in the `example/build`
+folder.
 
-Run
----
+Run Example
+-----------
```
Comment (marcusschiesser) [inline README.md]:
```suggestion
Run Examples
-----------s
```

**Review by marcusschiesser — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -17,8 +17,8 @@ Build
 output will be in the `dist` folder. Examples will be in the `example/build`
 folder.
 
-Run Example
------------
+Run Examples
+-----------s
```
Comment (s25g5d4) [inline README.md]: extra s here

**Review by marcusschiesser — APPROVED** [review]
